### DISCUSSION TITLE
/zh-cn/blog/posts/devops_on_string

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#149] [Announcements]
# 绳子上的运维日记 - Reverier's Blog

这篇文章到底合不合规我也不清楚，大家看个乐子就行。为了防止本文变成某次护网或者蓄谋攻击的架构指导文章，我会混淆大部分实现细节。当然还是暴露了一些不影响安全的违规操作，在本文发布时已经全部进行了整改，请勿学习！

https://blog.woooo.tech/posts/devops_on_string/

**Comment by YunZh1Jun** [comment]
rx你好厉害🥰🥰🥰🥰🥰🥰🥰

**Comment by JBNRZ** [comment]
有点儿帅

**Comment by NoahBox** [comment]
rx你好厉害🥰🥰🥰🥰🥰🥰🥰
